nifi-registry-core/nifi-registry-web-ui/src/main/webapp/template.dev.html

```diff
@@ -19,7 +19,7 @@
 <html>
 <head>
     <title>NiFi Registry</title>
-    <base href="/">
+    <base href="./">
```

Comment:
I'm not sure if this change is required. What difference it makes?

Reply:
Knox adds a path fragment such as `/gateway/sandbox/nifi-registry-app/` in front of `/nifi-registry/`. If the `base href` is set to `/`, all relative resource references will be resolved without the Knox part.

For example, if a resource `/gateway/sandbox/nifi-registry-app/nifi-registry/images/x.png` is referenced with a relative path `images/x.png`, then `base href="/"` will resolve it as `/images/x.png` that will result a 404 error.

Reply:
This template is only used in UI-dev-mode. I don't think one would ever configure dev mode and knox together since dev mode is just the front end and proxies requests to a running back-end.

This change breaks dev-mode when you try to navigate directly (or refresh) to `http://localhost:18081/explorer/grid-list`

I don't think this change is needed.